lib/PPI/Statement/Include.pm

```diff
@@ -236,6 +236,37 @@ sub arguments {
 	return @args;
 }
 
+=head2 arguments
```

Comment:
typo in `=head2`

will be nice to have an example

Reply:
Thanks! Fixed the typo and added two examples. :)